winsup/utils/getprocaddr.c

```diff
@@ -43,22 +80,22 @@ ctrl_handler(DWORD ctrl_type)
 				 &address, NULL);
   if (count != 1)
     {
-      output (1, "Could not capture backtrace\n");
+    output (1, "Could not capture backtrace\n");    
```

Comment:
Let's not introduce white-space issues. (This not only applies here, but to similar changes further below.)

Reply:
I don't know how this came, maybe something is automatically formatting it? Anyway, I will change.

Reply:
Done https://github.com/msys2/msys2-runtime/pull/31/commits/585f456cde1115f5126efcef802db0ec19c739a6